file_picker/uploads/file_pickers.py

```diff
@@ -14,7 +16,8 @@ class Meta(object):
 
     def save(self, commit=True):
         image = super(ImageForm, self).save(commit=False)
-        file_path = self.cleaned_data['file']
+        # Strip any directory names from the filename
+        file_path = os.path.basename(self.cleaned_data['file'])
```

Comment:
Why do we need to strip the directory from the path now when we didn't before? And will this have any effect on fields with `upload_to` set?